src/app/app.component.html

```diff
@@ -1,4 +1,4 @@
-<app-navbar></app-navbar>
+<app-navbar *ngIf="(router.url != '/signIn') && (router.url != '/signUp')"></app-navbar>
 
 <br>
 
```

Comment:
If we're on the sign in or sign up page, don't show the navbar (most sign in/sign up pages don't have navbars).